src/System.Security.Cryptography.X509Certificates/src/Internal/Cryptography/Pal.Windows/Native/Interop.crypt32.cs

```diff
@@ -314,9 +314,6 @@ public static bool CertVerifyCertificateChainPolicy(ChainPolicy pszPolicyOID, Sa
         [DllImport(Libraries.Crypt32, CharSet = CharSet.Unicode, SetLastError = true)]
         public static extern bool CertFreeCertificateContext(IntPtr pCertContext);
```

Comment:
can you please similarly remove  this function too and use the implementation from the common ?

Reply:
Sure, that was the next one on my list, here you go: #31373 😄